src/dapla_metadata/variable_definitions/utils/constants.py

```diff
@@ -29,6 +30,9 @@
     "\n--- Machine generated fields. Do not edit ---\n"
 )
 
+VARDEF_DESCRIPTIONS_FILE_PATH = "src/dapla_metadata/variable_definitions/resources/vardef_model_descriptions_nb.yaml"
```

Comment:
The path should be loaded in through `config.py` with this value as a default, to follow the established pattern in the project.